Fix DockPanel and Marker image scaling issues 2

JoeRobich opened the pull request:
This is a continuation of an earlier PR https://github.com/fdorg/flashdevelop/pull/644

@100%
![image](https://cloud.githubusercontent.com/assets/611219/6953578/3f1c89fa-d898-11e4-9c18-036b9778938f.png)

@200
![image](https://cloud.githubusercontent.com/assets/611219/6953560/242bf9f0-d898-11e4-9f32-15e0e06e6b3e.png)

Meychi commented:
Hmm... i'm not satisfied how the icons end up looking. Is there anyway to improve the looks when they scale? Or should we create a bigger set of icons for the high dpi mode? Also in the latest commit the close icons (x) are scaled incorrectly at 100%.

This PR is ballooning atm so i might end up manually picking the changes.

JoeRobich commented:
I am guessing the Marker icons don't come out looking great because XPM doesn't support partial transparency. 

As for unintentional image scaling, I looked harder at the Quantizer code since that was something I was trusting to work properly. I found the following

```
// Draw the source image onto the copy bitmap,
// which will effect a widening as appropriate.
g.DrawImageUnscaled(source, bounds);
```

> The DrawImageUnscaled method draws an image using its physical size, so the image will have its correct size in inches regardless of the resolution (dots per inch) of the display device.

I squashed all the commits and changed that line to a `g.DrawImage`. I hope that resolves things. =)
